src/main/java/com/google/impactdashboard/Credentials.java

```diff
@@ -0,0 +1,104 @@
+package com.google.impactdashboard;
+
+import java.lang.RuntimeException;
+import com.google.impactdashboard.configuration.Constants;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.appengine.AppEngineCredentials;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ByteArrayInputStream;
+import java.util.concurrent.atomic.AtomicReference;
+
+/** 
+ * A class for retrieving the proper credentials for the current environment in 
+ * order to access the necessary APIs. 
+ */
+public class Credentials {
+
+  /** 
+   * Returns the proper credentials for the current environment. 
+   * @throws RuntimeException if credentials fail to be assigned.
+   */
+  public static GoogleCredentials getCredentials() {
+
+    AtomicReference<GoogleCredentials> credentials = new AtomicReference<GoogleCredentials>();
```

Comment:
Seems like this creates a new credentials each time. 

Whereas, it seems to me that this should be using a single credentials throughout the entire application. Is there something preventing this?

Reply:
If you would like, I can cache the credentials value so that it isn't recreated